tools/clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -7843,6 +7841,8 @@ def err_hlsl_load_from_mesh_out_arrays: Error<
    "output arrays of a mesh shader can not be read from">;
 def err_hlsl_out_indices_array_incorrect_access: Error<
    "a vector in out indices array must be accessed as a whole">;
+def err_hlsl_unsupported_long_vector: Error<
+   "Vectors of over 4 elements in %0 are not supported">;
```

Comment:
```suggestion
   "vectors of over 4 elements in %0 are not supported">;
```

Reply:
This change is not part of this PR, but a prerequisite as I tried to explain in this comment: https://github.com/microsoft/DirectXShaderCompiler/pull/7155#issuecomment-2672803480 I have fixed it in the relevant PR: #7143